doc/index.rst

```diff
@@ -62,3 +62,15 @@ as resources for more information.
    faq
    reliability
    more-info
+
+Binderhub in the wild
```

Comment:
BinderHub (and below). But should we call them "Binder deployments"?

Reply:
Not sure. I try and use Binderhub to talk about the software that we use to run mybinder.org and "a binder" for the thing you deploy on it. But I think in practice people use them interchangeably and we might confused things even more by using "binderhub" which isn't very common.